Libraries/LibMarkdown/Heading.h

```diff
@@ -35,15 +36,20 @@ namespace Markdown {
 
 class Heading final : public Block {
 public:
-    virtual ~Heading() override {}
+    Heading(Text&& m_text, int m_level)
+        : m_text(move(m_text))
```

Comment:
I don't think you need to `move()` it here, it's already a `Text&&`

Reply:
If I remove the `move`, it complains about calling a deleted constructor.